### ISSUE TITLE
timescale_sensitivity not found

### ISSUE THREAD

**Issue by euhruska** [markovmodel/msmtools#60]
I try to calculate timescale_sensitivity, butit doesn't find it. On the other side other commands works, liek timescales. I update the msmtools, but it still doesn't find it.

```
>>> msmtools.analysis.timescales(T)
>>> msmtools.analysis.timescale_sensitivity(T, 1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/eh22/local/lib/python2.7/site-packages/msmtools-1.1_2.g872cca9-py2.7-linux-ppc64.egg/msmtools/analysis/api.py", line 1699, in timescale_sensitivity
    return dense.sensitivity.timescale_sensitivity(T, k)
AttributeError: 'module' object has no attribute 'sensitivity'
```

**Comment by marscher** [comment]
fixed by #61

**Comment by marscher** [comment]
Thanks for spotting!
